# Introduction

Introduction

Spaceflight introduces unique environmental effects on biology. For example, the microgravity environment of spaceflight directly disrupts the normal gravitational signals organisms use on Earth as well causing secondary effects due to a loss of buoyancy-induced convection that then disrupts gas exchange and thermal regulatory systems. Such effects are made more complex by playing out against an increased exposure to ionizing radiation that has been difficult to replicate in terrestrial models. One powerful approach to start to dissect these diverse responses has been through transcriptome profiling and one group of organisms where these analyses have proven particularly informative has been the plants. Thus, multiple microarray- and RNAseq-based analyses of plants grown in spaceflight are now available. These are mostly focused on the model organism _Arabidopsis thaliana_, providing a rich data landscape within which to make comparisons. A combination of their individual analyses by the researchers performing the analyses and meta-analyses of the diverse experimental results have led to some important insights into shared plant responses to the spaceflight environment. For example, dissecting these transcriptional patterns, Choi et al., (2019) previously showed that part of the spaceflight response involved changes to the heat shock and cold inducible pathways as potential consequences of changes to oxidative phosphorylation that resemble the high light response. Similarly,...... cell walls……. Previous meta-analysis has discussed how these conditions can lead to anoxic regions around plant tissues, changes in cooling patterns, and increased radiation exposure (Barker et al., 2023, Meyers and Wyatt 2023, Hughes and Kiss 2022). Despite these advances in our understanding of plant response to spaceflight, the transcriptome data remains complex to analyze. This challenge arises from the diversity in experimental design between studies, the complexity of the environment, with stressors often poorly defined, and the difficulty in using transcriptional fingerprints drawn from terrestrial analyses to describe processes in the unique and very different environment of spaceflight. It is likely these transcriptional fingerprints are present in the data but are obscured to varying degrees by these other factors.

#### Background on spaceflight and its effects on plants <a href="#stu98sgs9072" id="stu98sgs9072"></a>

Understanding _Arabidopsis thaliana_ and its responses to spaceflight requires incorporation of transcriptomes, proteomes, metabolomes, genomes, and epigenetic modifications. The GeneLab data repository from NASA has compiled a vast collection of omics datasets in space biology omics data related to spaceflight (Berros et al., 2021). These datasets are crucial to helping us understand how plants might respond during long-duration missions and are particularly important when considering colonization plans that rely on plants as a component of bioregenerative life support for the crew. These analyses are focused on modeling using the model plant Arabidopsis and are accompanied by comprehensive metadata that cover various aspects of each experiment's design.

#### The GeneLab AWG transcriptional matrix and metadata <a href="#id-9bza7s2xmtm3" id="id-9bza7s2xmtm3"></a>

To facilitate comparisons between studies with similar experimental designs, we imported 15 plant transcriptome datasets related to spaceflight from the GeneLab repository using the GeneLab API (Berrios et al., 2021) into an Arabidopsis transcriptional dataframe. We then attached the GeneLab Plant Analysis Working Groups manually curated metadata Matrix (referred to as “The Matrix” from Barker et al., 2023). This dataframe and metadata matrix only contains a fraction of the GeneLab metadata but allows for more robust comparisons between studies sharing commonalities in the primary treatment experimental design. Previous meta-analyses using the Matrix, Barker et al ., (2023) used DESeq2 linear differential expression models to identify spaceflight-related changes in cell wall processes and oxidative stress. Furthermore, their analysis revealed new response elements such as conserved changes in expression of genes and shifts in mono and divalent cation transport that are triggered by spaceflight related treatments. Barker et al., (2023) discuss how their statistical analysis of the Matrix was unable to deconvolute the tissue and hardware factors due to small sample sizes. It also identified factors within experimental design, such as flight hardware choice, preservation method and assay technique, that may introduce greater variation between datasets than the actual spaceflight treatment itself. The Matrix provides researchers with a powerful tool to explore transcriptomic data generated during spaceflight-related studies in plant biology and highlights new opportunities for additional experiments, controls and analysis methods such as machine learning (ML).

#### Machine learning and its applications in biology <a href="#id-6z4qk69hfddu" id="id-6z4qk69hfddu"></a>

Machine learning (ML) is an emergent field that has the potential to provide powerful new insights into modern and legacy archived data. Wang et al., (2018) showed that the use of ML increased accuracy and sensitivity of identifying differentially expressed genes (DEGs) in RNA-seq data. They assessed the performance of various feature selection algorithms and classification methods to predict DEGs involved in ethylene signaling in Arabidopsis (Want et al., 2018). The top 23 most informative features were identified using ‘InfoGain feature’ selection combined with Logistic Regression classification, demonstrating a powerful prediction capabilities for identification of important transcripts from RNAseq. Another study by Holloway _et al.,_ (2007) also used a combination of machine learning combined with traditional RNA-seq analysis methods to greatly improve the sensitivity of differentially expressed genes identification. Their integration of data about known association of TFs and their DNA binding site sequences, allowed them to make new predictions about potential TF target sites in the yeast genome. Eslami, _et al.,_ (2023) showed that both their ML algorithms were able to predict Autism spectrum disorder (ASD) “when all datasets are combined” potentially providing new tools to diagnose both child and adult patients. Taken together Machine learning techniques combined with detailed metadata about the environmental factors and additional genotypic details from subject matters can be used to identify new genetic response operons at a range of developmental stages and species.

### &#x20;<a href="#id-7t6bwfdcno1r" id="id-7t6bwfdcno1r"></a>

#### Results: Machine learning analysis spaceflight and related stressors <a href="#ckv2tbyl183j" id="ckv2tbyl183j"></a>

[Results from JADBIO](https://app.jadbio.com/share/0eec95b8-95c3-481a-bee7-e742b96c61ab)

[Supplementary table](https://docs.google.com/spreadsheets/d/1f\_0TuJROgpQNYvMKF2Kr69cQbWZGzSNypwZlvdiX5AY/edit?usp=sharing)

