---
description: >-
  Richard Barker, Lauren M. Sanders, Giorgos Papoutsoglou, Christina M. Johnson
  and Simon Gilroy
---

# Cover Page

_Title:_ **Identifying Genetic Response of Arabidopsis to Spaceflight Using Machine Learning**



_DRB Orcid ID:_ [_https://orcid.org/0000-0001-5681-9857_](https://orcid.org/0000-0001-5681-9857)

_Affiliations: NASA GeneLab and Blue Marble Space Institute of Science._

_UW-Madison, Botany Department._



**Abstract**

Biology reacts to the unique environment of spaceflight through diverse response networks ranging from shifts in physiology and biochemistry to exhibiting new, and often complex, patterns of gene expression. Although transcriptomic profiling has proven a powerful tool to dissecting these reactions to life in space, extracting the fingerprints of specific response systems has proven complex. This challenge arises because spaceflight stressors are diverse, leading to patterns of transcription that are equally diverse. Further, these changes in gene expression are themselves played out against biological responses to unique environmental backgrounds such as the increased exposure to ionizing radiation or altered gravitational environment inherent in spaceflight. These backgrounds are simply not present in the terrestrial datasets that are used to define and so extract the transcriptional characteristics of responses. This study investigates the impact of spaceflight on plants but utilizes machine learning techniques to more clearly identify genetic features associated with specific treatments. The study employed transcriptome datasets and preprocessing available through the GeneLab data repository but then applied further analysis using the JADBio Machine Learning platform. Three thousand and seventeen machine learning models were run as part of the training algorithm to generate a treatment model capable of identifying genetic features in the spaceflight data that were linked to treatment factors such as exposure to microgravity. The associated KEGG and GO analyses revealed both well-recognized and new plant molecular responses related to stressors from spaceflight. For example, this analysis reinforced conclusions from previous analyses of spaceflight-associated alterations in plastid function, but now highlighting chloroplast primary carbon metabolism as a principal target. The machine learning approach also revealed insights into novel response networks, such as spaceflight’s likely effect on plasmodesmatal function. These findings contribute significantly to our understanding of how plants respond when exposed to spaceflight and demonstrate that machine learning techniques can greatly enhance our ability to efficiently analyze the complex multifactorial datasets inherent in studies of biology in space.

**Keywords:** _Arabidopsis thaliana_; machine learning, plasmodesmata, auxin, cytokinin, cell wall, elevated CO2, mechanics, morphogenesis, roots, shoots, gravity, radiation, fixation, oxidative stress, chloroplasts, mitochondria, respiration, photosynthesis, ERAD, CHORAD, PDAD,



### Data set: “Final\_MetaAndTranscriptData\_DedupByMedian\_4JADBio\_v3” <a href="#if5ssizahdct" id="if5ssizahdct"></a>

[All factors at replicate level:](https://static-content.springer.com/esm/art%3A10.1038%2Fs41526-023-00247-6/MediaObjects/41526\_2023\_247\_MOESM2\_ESM.xlsx) from Barker et al., (2023)

{% embed url="https://static-content.springer.com/esm/art%3A10.1038%2Fs41526-023-00247-6/MediaObjects/41526_2023_247_MOESM2_ESM.xlsx" %}
