# References

**References (\~53+ so far)**

Alcázar, Bueno, and Tiburcio. Polyamines: Small Amines with Large Effects on Plant Abiotic Stress Tolerance. Cells. 2020 Oct 29;9(11):2373.

[http://doi:10.3390/cells9112373](about:blank)

Bates, Rosenthal, Sun, Chattopadhyay, Peffer, Yang, Ort, and Jones (2012). A Comparative Study of the Arabidopsis thaliana Guard-Cell Transcriptome and Its Modulation by Sucrose. PLoS ONE 7(11): e49641.

[https://doi.org/10.1371/journal.pone.0049641](https://doi.org/10.1371/journal.pone.0049641)

Band (2021). “Auxin fluxes through plasmodesmata”. New Phytol, 231: 1686-1692. [https://doi.org/10.1111/nph.17517](https://doi.org/10.1111/nph.17517)

Baqué, Hanke, Böttger, Leya, Moeller, and Vera (2018). Protection Of Cyanobacterial Carotenoids' Raman Signatures By Martian Mineral Analogues After High-dose Gamma Irradiation. Journal of Raman Spectroscopy, 10(49), 1617-1627.

[https://doi.org/10.1002/jrs.5449](https://doi.org/10.1002/jrs.5449)

Barbez, Dünser, Gaidora, Lendl, and Busch (2017). Auxin steers root cell expansion via apoplastic pH regulation in _Arabidopsis thaliana_. Proc Natl Acad Sci USA. Jun 13;114(24):E4884-E4893.

[https://doi:10.1073/pnas.1613499114](about:blank).

Barker, Lombardino, Rasmussen and Gilroy (2020). Test of Arabidopsis Space Transcriptome: A Discovery Environment to Explore Multiple Plant Biology Spaceflight Experiments. Frontiers in Plant Science.

[https://doi.org/10.3389/fpls.2020.00147](https://doi.org/10.3389/fpls.2020.00147)

Barker, Kruse, Johnson, Saravia-Butler, Fogle, Chang, Trane, Kinscherf, Villacampa, Manzano, Herranz, Davin, Lewis, Perera, Wolverton, Gupta, Jaiswal, Reinsch, Wyatt, and Gilroy_._ (2023). Meta-analysis of the space flight and microgravity response of the Arabidopsis plant transcriptome. _npj Microgravity_ 9, 21.

[https://doi.org/10.1038/s41526-023-00247-6](https://doi.org/10.1038/s41526-023-00247-6)

Barker, Costes, Miller, Gebre, Lombardino and Simon[ ](https://www.nature.com/articles/s41526-021-00143-x#auth-Simon-Gilroy-Aff1)Gilroy (2021). Rad-Bio-App: a discovery environment for biologists to explore spaceflight-related radiation exposures. _npj Microgravity_ 7, 15. [https://doi.org/10.1038/s41526-021-00143-x](https://doi.org/10.1038/s41526-021-00143-x)

Brady,[ ](https://www.science.org/doi/full/10.1126/science.1146265#con2)Orlando, Lee, Wang, Koch, Dinneny, Mace, Ohler and Benfey (2007). A High-Resolution Root Spatiotemporal Map Reveals Dominant Expression Patterns. Science. 318,801-806(2007).

[https://DOI:10.1126/science.1146265](about:blank)

Berrios Galazka, Grigorev, Gebre, and Costes (2021). NASA GeneLab: interfaces for the exploration of space omics data, Nucleic Acids Research, Volume 49, Issue D1, 8 January, Pages D1515–D1522, [https://doi.org/10.1093/nar/gkaa887](https://doi.org/10.1093/nar/gkaa887)

Campos, Dressano, Ceciliato, Guerrero-Abad, Silva, Fiori, Morato do Canto, Bergonci, Claus, Silva-Filho and Moura (2018). Arabidopsis thaliana rapid alkalinization factor 1–mediated root growth inhibition is dependent on calmodulin-like protein 38. Journal of Biological Chemistry,Volume 293, Issue 6, Pages 2159-2171, ISSN 0021-9258,

[https://doi.org/10.1074/jbc.M117.808881](https://doi.org/10.1074/jbc.M117.808881).

Carolin, Quint, and Wigge (2022). Recent advances in understanding thermomorphogenesis signaling. Current Opinion in Plant Biology,

Volume 68, 102231, ISSN 1369-5266,

[https://doi.org/10.1016/j.pbi.2022.102231](https://doi.org/10.1016/j.pbi.2022.102231).

Cools, Toon, Iantcheva, Weimer, Boens, Takahashi, Maes, Van den Daele, Isterdael, Schnittger, and De Veylder. (2011) “The ‘_Arabidopsis thaliana’_ Checkpoint Kinase WEE1 Protects against Premature Vascular Differentiation during Replication Stress.” The Plant Cell 23, no. 4 : 1435–48.

[http://www.jstor.org/stable/41433401](http://www.jstor.org/stable/41433401)

Choi, Barker, Kim, Swanson, and Gilroy (2019). Variation in the transcriptome of different ecotypes of Arabidopsis thaliana reveals signatures of oxidative stress in plant responses to spaceflight. American Journal of Botany.

[https://DOI:10.1002/ajb2.1223](about:blank)

De Clercq, Vermeirssen, Van Aken, Vandepoele, Murcha, Law, Inze, Ng, Ivanova, Rombaut, van de Cotte, Jaspers, Van de Peer, Kangasjärvi, Whelan, and Frank Van Breusegem (2013). The membrane-bound NAC transcription factor ANAC013 functions in mitochondrial retrograde regulation of the oxidative stress response in Arabidopsis. Plant Cell 25: 3472–3490

[http://doi:10.1105/tpc.113.117168](about:blank)

Engelhorn, Julia, Blanvillain, Kröner, Parrinello, Rohmer, Posé, Ott, Schmid, and Carles. (2017). "Dynamics of H3K4me3 Chromatin Marks Prevails over H3K27me3 for Gene Regulation during Flower

Morphogenesis in _Arabidopsis_ _thaliana_" Epigenomes 1, no. 2: 8. [https://doi.org/10.3390/epigenomes1020008](https://doi.org/10.3390/epigenomes1020008)

Farooq, Tehseen, Sabir, and Atal (2023). Detection of autism spectrum disorder (ASD) in children and adults using machine learning. Sci Rep 13, 9605.

[https://doi.org/10.1038/s41598-023-35910-1](https://doi.org/10.1038/s41598-023-35910-1)

Fan, Liao, Tan, Wang, Schroeder and Li (2023). Arabidopsis PLANT U-BOX44 down-regulates osmotic stress signaling by mediating Ca2+-DEPENDENT PROTEIN KINASE4 degradation. Plant Cell. Jun 20:koad173.

[http://doi:10.1093/plcell/koad173](about:blank).

Gámez-Arjona, Sánchez-Rodríguez and Montesinos (2022). The root apoplastic pH as an integrator of plant signaling. Front Plant Sci. Aug 23;13:931979.

[https://doi:10.3389/fpls.2022.931979](about:blank).

Guo, Tingwei, Kong, Burton, Scaglione, Beagles, Ray, and Ramonell (2022). "The E3 Ubiquitin Ligase ATL9 Affects Expression of Defense Related Genes, Cell Death and Callose Deposition in Response to Fungal Infection". Pathogens 11, no. 1: 68.

[https://doi.org/10.3390/pathogens11010068](https://doi.org/10.3390/pathogens11010068)

Gutzat, Borghi, Fütterer, Bischof, Laizet, Hennig, Feil, Lunn, and Gruissem (2011). RETINOBLASTOMA-RELATED PROTEIN controls the transition to autotrophic plant development. Development; 138 (14): 2977–2986.

[https://doi.org/10.1242/dev.060830](https://doi.org/10.1242/dev.060830)

Gull, Lone and Wani (2019). “Biotic and Abiotic Stresses In Plants”. [https://doi.org/10.5772/intechopen.85832](https://doi.org/10.5772/intechopen.85832)

Ge, Jung and Yao (2020). ShinyGO: a graphical gene-set enrichment tool for animals and plants. Bioinformatics. 36(8):2628-2629.

[https://doi:10.1093/bioinformatics/btz931](about:blank)

Johnson, Subramanian, Pattathil, Correll, and Kiss (2017). Comparative transcriptomics indicate changes in cell wall organization and stress response in seedlings during spaceflight. _Am. J. Bot._ 104, 1219–1231

[http://doi:10.3732/ajb.1700079](about:blank)

Kacprzak, Dahlqvist, and Van Aken. (2020). The transcription factor ANAC017 is a key regulator of mitochondrial proteotoxic stress responses in plants. Philos Trans R Soc Lond B Biol Sci. Jun 22;375(1801):20190411.

[http://doi:10.1098/rstb.2019.0411](about:blank)

Paul, Haveman, Califar, and Ferl (2021). Epigenomic regulators elongator complex subunit 2 and methyltransferase 1 differentially condition the spaceflight response in Arabidopsis. Frontiers in Plant Science. 12691790.

[https://doi.org/10.3389/fpls.2021.691790](https://doi.org/10.3389/fpls.2021.691790)

Hassani-Pak, Singh, Brandizi, Hearnshaw, Parsons, Amberkar, Phillips, Doonan, and Rawlings (2021). Knetminer: a comprehensive approach for supporting evidence-based gene discovery and complex trait analysis across species. Plant Biotechnology Journal 19:1670–1678.

[https://doi.org/10.1111/pbi.13583](https://doi.org/10.1111/pbi.13583)

He, Berkowitz, Hu, Zhao, Qian, Shou, Whelan, Wang and Wang (2021). Co-regulation of mitochondrial and chloroplast function: Molecular components and mechanisms

[https://doi.org/10.1016%2Fj.xplc.2022.100496](https://doi.org/10.1016%2Fj.xplc.2022.100496)

Holloway, Kon, and DeLisi (2007). Machine learning for regulatory analysis and transcription factor target prediction in yeast. Syst Synth Biol 1, 25–46.

[https://doi.org/10.1007/s11693-006-9003-3](https://doi.org/10.1007/s11693-006-9003-3)

Kendall, Hellwege, Marriot, Whalley, Graham, and Penfield (2011). Induction of dormancy in Arabidopsis summer annuals requires parallel regulation of DOG1 and hormone metabolism by low temperature and CBF transcription factors. Plant Cell. 2011 Jul;23(7):2568-80.

[https://doi:10.1105/tpc.111.087643](about:blank)

Kruse, Meyers, Basu, Hutchinson, Luesse & Wyatt (2020). Spaceflight induces novel regulatory responses in Arabidopsis seedling as revealed by combined proteomic and transcriptomic analyses. BMC Plant Biol 20, 237.

[https://doi.org/10.1186/s12870-020-02392-6](https://doi.org/10.1186/s12870-020-02392-6)

Lai, Doherty, Mueller-Roeber, Kay, Schippers, and Dijkwel (2012). CIRCADIAN CLOCK-ASSOCIATED 1 regulates ROS homeostasis and oxidative stress responses. PNAS. 109 (42). 17129-17134

[https://doi.org/10.1073/pnas.1209148109](https://doi.org/10.1073/pnas.1209148109)

Leivar and Monte (2014). PIFs: systems integrators in plant development. Plant Cell. Jan;26(1):56-78.

[https://doi.org/10.1105/tpc.113.120857](https://doi.org/10.1105/tpc.113.120857)

Liu, Jinyi, Rice, Chen, Baum and Hewezi (2014). “Synchronization of Developmental Processes and Defense Signaling by Growth Regulating Transcription Factors.” PLoS ONE 9 (2014): n. Pag.

https://doi:[10.1371/journal.pone.0098477](https://doi.org/10.1371%2Fjournal.pone.0098477)

Liang, Li, Zhou, Li, Liu, Hao, Wang, Zhao, and Han (2015). Subcellular Distribution of NTL Transcription Factors in Arabidopsis thaliana. Traffic. Oct;16(10):1062-74.

[https://doi:10.1111/tra.12311](about:blank)

Lopez-Anido, Vatén, Smoot, Sharma, Guo, Gong, Anleu Gil, Weimer, and Bergmann (2021). Single-cell resolution of lineage trajectories in the Arabidopsis stomatal lineage and developing leaf. Dev Cell. 5;56(7):1043-1055.e4.

[https://doi:10.1016/j.devcel.2021.03.014](about:blank)

Liu, Strauss, Adibi, Mosca, Yoshida, Dello Ioio, Runions, Andersen, Grossmann, Huijser, Smith, and Tsiantis (2022). “Cytokinin promotes growth cessation in the Arabidopsis root”. Current Biology, Volume 32, Issue 9, Pages 1974-1985.e3.

[https://doi.org/10.1016/j.cub.2022.03.019](https://doi.org/10.1016/j.cub.2022.03.019)

Luo, and Brouwer (2013). Pathview: an R/Bioconductor package for pathway-based data integration and visualization, Bioinformatics, Volume 29, Issue 14, July 2013, Pages 1830–183. [https://doi.org/10.1093/bioinformatics/btt285](https://doi.org/10.1093/bioinformatics/btt285)

Kanehisa, Furumichi, Sato, Ishiguro-Watanabe, and Tanabe (2021). KEGG: integrating viruses and cellular organisms, Nucleic Acids Research, Volume 49, Issue D1, Pages D545–D551.

[https://doi.org/10.1093/nar/gkaa970](https://doi.org/10.1093/nar/gkaa970)

Kwon, Sparks, Nakashima, Allen, Tang, and Blancaflor (2015). Transcriptional response of Arabidopsis seedlings during spaceflight reveals peroxidase and cell wall remodeling genes associated with root hair development. _Am. J. Bot._ 102, 21–35.

[https://doi:10.3732/ajb.1400458](about:blank)

Overbey, Saravia-Butler, Zhang, Rathi, Fogle, da Silveira, Barker, Bass, Beheshti, Berrios, Blaber, Cekanaviciute, Costa, Davin, Fisch, Gebre, Geniza, Gilbert, Gilroy,Hardiman, Herranz, Kidane, Kruse, Lee, Liefeld, Lewis, McDonald, Meller, Mishra, Perera, Ray, Reinsch, Rosenthal, Strong,. Szewczyk, Tahimic, Taylor, Vandenbrink, Villacampa, Weging, Wolverton, Wyatt, Zea, Costes and Galazka. (2021) NASA GeneLab RNA-Seq Consensus Pipeline: Standardized Processing of Short-Read RNA-Seq Data. iScience.

[https://doi.org/10.1016/j.isci.2021.102361](https://doi.org/10.1016/j.isci.2021.102361)

Liu, Zhang, Zhao, Feng, Li, Yang, Luan, Li, and He (2013). Auxin controls seed dormancy through stimulation of abscisic acid signaling by inducing ARF-mediated ABI3 activation in Arabidopsis. Proc Natl Acad Sci U S A. 2013 Sep 17;110(38):15485-90.

[https://doi:10.1073/pnas.1304651110](about:blank).

Liang, Gao, Wang, Zhang, Yin, Chen, Zhang, and Zhao (2020). Phytochrome-interacting factors regulate seedling growth through ABA signaling, Biochemical and Biophysical Research Communications,Volume 526, Issue 4,Pages 1100-1105, ISSN 0006-291.

[https://doi.org/10.1016/j.bbrc.2020.04.011](https://doi.org/10.1016/j.bbrc.2020.04.011)

Ludovici, Chierici, Souza, d'Errico, Iannotti, and Malizia (2022). “Effects Of Ionizing Radiation On Flora Ten Years After the Fukushima Dai-ichi Disaster”. Plants, 2(11), 222. [https://doi.org/10.3390/plants11020222](https://doi.org/10.3390/plants11020222)

Piya, Liu, Burch-Smith , Baum , and Hewezi (2020). A role for Arabidopsis growth-regulating factors 1 and 3 in growth-stress antagonism. J Exp Bot. Feb 19;71(4):1402-1417.

[https://doi:10.1093/jxb/erz502](about:blank)

Piya, Shrestha, Binder, Stewart, and Hewezi (2014). Protein-protein interaction and gene co-expression maps of ARFs and Aux/IAAs in Arabidopsis. Front Plant Sci. 2014 Dec 23;5:744.

[https://doi:10.3389/fpls.2014.00744](about:blank).

Roudier, Ahmed, Bérard, Sarazin, Mary-Huard, Cortijo, Bouyer, Caillieux, Duvernois-Berthet, Al-Shikhley, Giraut, Després, Drevensek, Barneche, Dèrozier, Brunaud, Aubourg, Schnittger, Bowler, Martin-Magniette, Robin and Colot (2011). “Integrative epigenomic mapping defines four main chromatin states in Arabidopsis”. EMBO J. May 18;30(10):1928-38.

[https://doi:10.1038/emboj.2011.103](about:blank)

Shanmugabalaji, Chahtane, Accossato, Rahire, Gouzerh, Lopez-Molina and Kessler (2018). Chloroplast Biogenesis Controlled by DELLA-TOC159 Interaction in Early Plant Development. Current Biology, Volume 28, Issue 16, 2616 - 2623.

https:/doi:[10.1016/j.cub.2018.06.006](https://doi.org/10.1016/j.cub.2018.06.006)

Shanmugabalaji and Kessler (2019). CHLORAD: Eradicating Translocon Components from the Outer Membrane of the Chloroplast. Mol Plant. Apr 1;12(4):467-469.

[https://doi:10.1016/j.molp.2019.03.002](about:blank).

Sun P, Wang H, Zhao P, Yu Q, He Y, Deng W, Guo H. The Regulation of Xylem Development by Transcription Factors and Their Upstream MicroRNAs. Int J Mol Sci. 2022 Sep 4;23(17):10134.

[https://doi:10.3390/ijms231710134](about:blank).

Sakakibara (2021). Cytokinin biosynthesis and transport for systemic nitrogen signaling. Plant J. Jan;105(2):421-430.

[https://doi:10.1111/tpj.15011](about:blank)

Sugimoto, Tanaka, Muramoto, Kitagawa-Yogo, and Mitsukawa (2022). Transcription factor NTL9 negatively regulates Arabidopsis vascular cambium development during stem secondary growth, Plant Physiology, Volume 190, Issue 3, November, Pages 1731–1746. [https://doi.org/10.1093/plphys/kiac368](https://doi.org/10.1093/plphys/kiac368)

Schikora, Carreri, Charpentier, and Hirt (2008). The Dark Side of the Salad: _Salmonella typhimurium_ Overcomes the Innate Immune Response of _Arabidopsis thaliana_ and Shows an Endopathogenic Lifestyle. PLOS ONE 3(5): e2279.

[https://doi.org/10.1371/journal.pone.0002279](https://doi.org/10.1371/journal.pone.0002279)

Serre, Wernerová, Vittal, Dubey, Medvecká, Jelínková, Petrášek, Grossmann, Fendrych (2023). The AUX1-AFB1-CNGC14 module establishes a longitudinal root surface pH profile eLife 12:e85193

https://doi.org/

Tack, Wamelink, Denkova, Schouwenburg, Hilhorst, Wolterbeek, and Goedhart (2021). “Influence Of Martian Radiation-like Conditions On the Growth Of Secale Cereale And Lepidium Sativum.” Frontiers in Astronomy and Space Sciences, (8).

[https://doi.org/10.3389/fspas.2021.665649](https://doi.org/10.3389/fspas.2021.665649)

Meyers, and Wyatt. (2023). Plant Space Biology in the Genomics Age. In Annual Plant Reviews online, J.A. Roberts (Ed.).

[https://doi.org/10.1002/9781119312994.apr0784](https://doi.org/10.1002/9781119312994.apr0784)

Nishimura, Mori, Shikata, Nakamura, Hashiguchi, Abe, Hagihara, Yoshikawa, Toyota, Higaki and Morita (2023 ). Cell polarity linked to gravity sensing is generated by LZY translocation from statoliths to the plasma membrane. Science. Sep;381(6661):1006-1010.

[https://doi:10.1126/science.adh9978](about:blank)

Takahashi, Inagaki, Nishimura, Sakakibara, Antoniadi, Karady, Ljung and Umeda (2020) DNA damage inhibits root growth by enhancing cytokinin biosynthesis in Arabidopsis thaliana. bioRxiv .160838.

[https://doi.org/10.1101/2020.06.19.160838](https://doi.org/10.1101/2020.06.19.160838)

Tsamardinos, Charonyktakis, Papoutsoglou, Borboudakis, Lakiotaki, Zenklusen, Juhl, Chatzaki, and Lagani (2022). Just Add Data: automated predictive modeling for knowledge discovery and feature selection. npj Precis. Onc. 6, 38.

[https://doi.org/10.1038/s41698-022-00274-8](https://doi.org/10.1038/s41698-022-00274-8)

Ma, Yan, Yang, Liu, Li, Sheng, Cao, Yu, Yi, Xu, and Su (2021). PlantGSAD: a comprehensive gene set annotation database for plant species. Nucleic Acids Research.

[https://doi:10.1093/nar/gkab794](about:blank)

Wang, Xi, Sung, and Qiao (2018). RNA-seq assistant: machine learning based methods to identify more transcriptional regulated genes. BMC Genomics 19, 546.

[https://doi.org/10.1186/s12864-018-4932-2](https://doi.org/10.1186/s12864-018-4932-2)

Wang, Chen, Lam, Dini-Andreote, Dai & Zhong Wei. (2022).Multifaceted roles of flavonoids mediating plant-microbe interactions. _Microbiome_ 10, 233.

[https://doi.org/10.1186/s40168-022-01420-x](https://doi.org/10.1186/s40168-022-01420-x)

Xu, Jiang and Zhou (2015). Response and adaptation of photosynthesis, respiration, and antioxidant systems to elevated CO2 with environmental stress in plants. Front Plant Sci. Sep 10;6:701.

https://doi: 10.3389/fpls.2015.00701

Yeats and Rose (2013). The Formation and Function of Plant Cuticles, Plant Physiology, Volume 163, Issue 1, September 2013, Pages 5–20,

[https://doi.org/10.1104/pp.113.222737](https://doi.org/10.1104/pp.113.222737)

Zhou, Zhou, Pache, Chang, Khodabakhshi, Tanaseichuk, Benner, and Chanda (2019). “Metascape provides a biologist-oriented resource for the analysis of systems-level datasets”. Nature Communications. 10(1):1523.

[https://doi.org/10.1038/s41467-019-09234-6](https://doi.org/10.1038/s41467-019-09234-6)

Zupanska, Denison, Ferl and Paul (2013). “Spaceflight engages heat shock protein and other molecular chaperone genes in tissue culture cells of _Arabidopsis thaliana_”. Am J Bot. Jan;100(1):235-48.

[https://doi:10.3732/ajb.1200343](about:blank)

Zhang, Vancea, Shahul, Hameed and Arold (2021). Versatile control of the CDC48 segregase by the plant UBX-containing (PUX) proteins. Comput Struct Biotechnol J. May 14;19:3125-3132. [https://doi:10.1016/j.csbj.2021.05.025](about:blank).

Zhu, Yu, Wang, Zhao, Li, Fan, Shang, Du, Wang, Wu, Xu, Zhang, and Zhang (2007). Two calcium-dependent protein kinases, CPK4 and CPK11, regulate abscisic acid signal transduction in Arabidopsis. Plant Cell. (10):3019-36.

[http://doi:10.1105/tpc.107.050666](about:blank).

Leivar, P., & Monte, E. (2014). PIFs: Systems Integrators in Plant Development. _The Plant Cell,_ 26(1), 56-78. [https://doi.org/10.1105/tpc.113.120857](https://doi.org/10.1105/tpc.113.120857)

Leivar, P. & Monte, E. (2014). PIFs: Systems Integrators in Plant Development. _Plant Cell_, 26(1), 56-78. [https://doi.org/10.1105/tpc.113.120857](https://doi.org/10.1105/tpc.113.120857)

Chew, Y. H., Wenden, B., Flis, A., Mengin, V., Taylor, J., Davey, C. L., Tindal, C., Thomas, H., Ougham, H. J., De Reffye, P., Stitt, M., Williams, M., Muetzelfeldt, R., Halliday, K. J., & Millar, A. J. (2014). Multiscale digital _Arabidopsis_ predicts individual organ and whole-organism growth. _Proceedings of the National Academy of Sciences_, _111_(39), E4127-E4136. https://doi.org/10.1073/pnas.1410238111

Chew, Y. H., Wenden, B., Flis, A., Mengin, V., Taylor, J., Davey, C. L., Tindal, C., Thomas, H., Ougham, H. J., De Reffye, P., Stitt, M., Williams, M., Muetzelfeldt, R., Halliday, K. J., & Millar, A. J. (2014). Multiscale digital _Arabidopsis_ predicts individual organ and whole-organism growth. _Proceedings of the National Academy of Sciences_, _111_(39), E4127-E4136. https://doi.org/10.1073/pnas.1410238111

